extensions/data-plane-transfer/data-plane-transfer-sync/src/main/java/org/eclipse/dataspaceconnector/transfer/dataplane/sync/DataPlaneTransferSyncExtension.java

```diff
@@ -69,30 +74,46 @@ public String name() {
 
     @Override
     public void initialize(ServiceExtensionContext context) {
-        registerValidationApi(context);
+        var privateKeyAlias = context.getConfig().getString(TOKEN_SIGNER_PRIVATE_KEY_ALIAS);
+        var publicKeyAlias = context.getSetting(TOKEN_VERIFIER_PUBLIC_KEY_ALIAS, privateKeyAlias + "-pub");
+        var proxyEndpoint = context.getConfig().getString(DATA_PROXY_ENDPOINT); // TODO: this should determined dynamically for every DataRequest through DataPlaneSelector
```

Comment:
`context.getConfig()` could be extracted in a variable and all the settings obtained from it.